bitbots_body_behavior/src/bitbots_body_behavior/actions/kick_ball.py

```diff
@@ -69,7 +70,18 @@ def perform(self, reevaluate=False):
                     goal.ball_position.y = 0.0
                     goal.ball_position.z = 0
                     goal.unstable = True
-                    kick_direction = math.radians(25)
+
+                    # only check 2 directions, left and right
+                    kick_directions = sorted(np.linspace(
+                        -self.penalty_kick_angle,
+                        self.penalty_kick_angle,
+                        num=2), key=abs)
```

Comment:
```suggestion
                    kick_directions = np.array([-self.penalty_kick_angle, self.penalty_kick_angle])
```
I think this change would reduce unnecessary code complexity here.